stdlib/system/src/main/ballerina/src/system/system.bal

```diff
@@ -14,27 +14,56 @@
 // specific language governing permissions and limitations
 // under the License.
 
+import ballerinax/java;
 
 # Returns the environment variable value associated with the provided name.
 #
 # + name - Name of the environment variable
 # + return - Environment variable value if it exists, else an empty string
-public function getEnv(@untainted string name) returns string = external;
+public function getEnv(@untainted string name) returns string {
+    return covertToString(nativeGetEnv(java:fromString(name)));
+}
+
+function nativeGetEnv(handle key) returns handle = @java:Method {
+    name: "getEnv",
+    class: "org.ballerinalang.stdlib.system.nativeimpl.GetEnv"
```

Comment:
We can use the `java.lang.System.getEnv()` for this, no need to use another class here, since what the `GetEnv` class only does is to call that particular method.

It will return the string as a `handle` and we can convert it to ballerina `string`.

Handle the `nil` value properly.

You can see how to use `java.lang` methods in inter-op [here](https://github.com/ballerina-platform/ballerina-lang/blob/03ffe281f821bc43f8dfd79f1d80df0d45764004/stdlib/stringutils/src/main/ballerina/src/stringutils/natives.bal#L166).

Please check for similar cases.